2022/0xxx/CVE-2022-0715.json

```diff
@@ -90,9 +90,14 @@
     "references": {
```

Comment:
('Expecting property name enclosed in double quotes: line 93 column 1 (char 4679)',)